CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+<!-- TOC -->
+
+- [Filing Bugs](#filing-bugs)
+- [Code Style](#code-style)
+- [Core Values](#core-values)
+- [Writing Tests](#writing-tests)
+
+<!-- /TOC -->
+
+Thanks for taking the time to contribute! 🎉
+
+HyperApp is an open source project and we love to receive contributions from our community. You can start to contribute in many ways, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this project, but don't be afraid to break any rules. Use your best judgment, and feel free to propose changes in a pull request.
+
+All members of our community are expected to follow our [Code of Conduct](/CODE_OF_CONDUCT.md), please be welcoming and friendly in all your interactions with the project.
+
+## Filing Bugs
+
+- Before submitting a bug report, search the issues for similar tickets. Your issue may have already been discussed resolved. Feel free to add a comment to an existing ticket, even if it's closed.
+
+- Determine which repository the problem should be reported in. If you have an issue with the website, you'll be better served in [hyperapp/website](https://github.com/hyperapp/website), etc. If you would like to share something cool you've made with HyperApp, check out [hyperapp/awesome](https://github.com/hyperapp/awesome-hyperapp).
+
+- If you have a question or need help with something you are building, you'll get faster results if you can join the [HyperApp Slack Team](https://hyperappjs.herokuapp.com).
```

Comment:
suggestion:

~you'll get faster...~  => "we recommend joining the HyperApp Slack Team, and asking in the #learning channel